src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl_IVsSolutionEvents.cs

```diff
@@ -92,8 +92,6 @@ int IVsSolutionEvents.OnAfterCloseSolution(object pUnkReserved)
 
         internal static bool IsDeferredSolutionLoadEnabled(IServiceProvider serviceProvider)
         {
-            // NOTE: It is expected that the "as" will fail on Dev14, as IVsSolution7 was
-            // introduced in Dev15.  Be sure to handle the null result here.
             var solution7 = serviceProvider.GetService(typeof(SVsSolution)) as IVsSolution7;
```

Comment:
Should this hard cast then, since we don't expect that to fail again? It'll eliminate some ?. on the next line too.